relnotes/reverse.feature.md

```diff
@@ -0,0 +1,5 @@
+* `ocean.core.array.Iteration`
+
+  New module containing single function for now - `reverse`, which allows to
+  iterate given slice in reverse order without any allocations (similar to
+  built-in `foreach_reverse`).
```

Comment:
Yes, it worked good enough for me, and it's less source code. I like that and take the liberty of boldly claiming it works good enough for every reverse array iteration.